base/compiler/inferencestate.jl

```diff
@@ -391,16 +403,17 @@ function compute_trycatch(code::Vector{Any}, ip::BitSet, bbs::Union{Vector{Basic
     # then we can find all `try`s by walking backwards from :enter statements,
     # and all `catch`es by looking at the statement after the :enter
     n = length(code)
-    empty!(ip)
+    ip = BitSet()
     ip.offset = 0 # for _bits_findnext
```

Comment:
Is this still needed now that you create a new BitSet?

Reply:
I'm not sure honestly. It looks like this optimization is trying to take advantage of `BitSet`'s implementation details. @vtjnash any thoughts on this?

Reply:
Oh I thought this might be the initialization value, but it turns out the init value is a sentinel (`NO_OFFSET`) and this is just specifying a reasonable value for the offset up-front.

In any case, I don't think the change in this PR affects this 👍